v0.7/mlperf.conf

```diff
@@ -45,8 +46,11 @@ ssd-resnet34.Server.target_latency = 100
 gnmt.Server.min_query_count = 90112
 gnmt.Server.target_latency = 250
 gnmt.Server.target_latency_percentile = 97
+rnnt.Server.min_query_count = 90112
+rnnt.Server.target_latency = 13000
```

Comment:
Does this mean a 13ms latency target for RNN-T in Server scenario?

Reply:
No, it's 13 seconds. This may seem incredibly large, and it is. Note that 13 seconds is the 70% input duration of samples in the dev-clean dataset. It's fairly hard to get an RNN-T model that moves much faster than 1x real-time factor. You could probably get a big speedup with persistent RNN kernels, but I'm developing on standard intel CPUs (not entirely sure what it is), and it's questionable whether you can actually fit all of the params into the register file for the prediction network, which is similar to a seq2seq model's decoder network (therefore, you cannot have the first RNN run completely, and then have the next RNN run completely, because it's autoregressive). It's complicated.

I'm not sure if you're aware of how persistent kernels work, but certainly others are not. But a different way to think about this is that the paper we're based off of https://arxiv.org/pdf/1811.06621.pdf achieves less than 2X real-time-factor (section 7.4) with a lot of engineering work (so a 13 second datapoint is processed in 6.5 seconds). So taking the outside view, we can expect that 1x real-time-factor may be somewhat reasonable.

Reply:
@galv @christ1ne Could we add the latency target discussion to tomorrow's agenda item? Thanks